crates/cli-support/src/js/binding.rs

```diff
@@ -813,12 +817,14 @@ fn instruction(js: &mut JsBuilder, instr: &Instruction, log_error: &mut bool) ->
             let func = js.cx.pass_to_wasm_function(kind.clone(), *mem)?;
             let malloc = js.cx.export_name_of(*malloc);
             let i = js.tmp();
+            let align = std::cmp::max(kind.size(), 4);
```

Comment:
I'm not sure why this is needed; I think it should be fine for the alignment to be less than 4.

Reply:
For some reason I was assuming that the minimum alignment was 4, which is obviously nonsense.